specs/bellatrix/beacon-chain.md

```diff
@@ -355,10 +355,17 @@ def verify_and_notify_new_payload(self: ExecutionEngine,
     """
     Return ``True`` if and only if ``new_payload_request`` is valid with respect to ``self.execution_state``.
     """
-    if not self.is_valid_block_hash(new_payload_request.execution_payload):
+    execution_payload = new_payload_request.execution_payload
+
+    if b'' in execution_payload.transactions:
```

Comment:
Having this check explicit makes sense, but i slightly prefer to generalize it to validate transactions encoding (including zero length check):

```python
if not self.is_valid_transaction_encoding(execution_payload.transactions):
    return False
```

Reply:
We currently don't have transaction types in the CL specs. We could add them (at the expense of losing opaque) but it's not necessary to fix the hash collision.